shared.csv

```diff
@@ -1 +1,24 @@
 name,quadrant,ring,isNew,description
+Kotlin,Languages & Frameworks,Trial,TRUE,
+TypeScript,Languages & Frameworks,Trial,TRUE,
+Spring Security,Languages & Frameworks,Trial,TRUE,
+Kibana,Tools,Trial,TRUE,
+Elasticsearch,Tools,Trial,TRUE,
+Filebeat,Tools,Trial,TRUE,
+Yarn,Tools,Trial,TRUE,
+Docker,Infrastructure Components,Trial,TRUE,
+GitHub Actions,Services,Trial,TRUE,
+Make,Tools,Trial,TRUE,
+Express,Languages & Frameworks,Trial,TRUE,
+Node.js,Languages & Frameworks,Trial,TRUE,
+React,Languages & Frameworks,Trial,TRUE,
+Redux,Languages & Frameworks,Trial,TRUE,
+Gradle,Tools,Trial,TRUE,
+Nimbus,Languages & Frameworks,Trial,TRUE,
+Postgresql,Infrastructure Components,Trial,TRUE,
+GraphQL,Languages & Frameworks,Assess,TRUE,At the moment there is no use case for our application but we'd like to use it at some point
+React i18next,Languages & Frameworks,Trial,TRUE,
+kustomize,Tools,Trial,TRUE,
+Kubernetes,Infrastructure Components,Trial,TRUE,
+Google CloudBuild,Services,Trial,TRUE,
```

Comment:
According to official page the naming is "Google Cloud Build" https://cloud.google.com/cloud-build

Reply:
Good catch, changed it!